specification/networkfunction/resource-manager/Microsoft.NetworkFunction/preview/2021-09-01-preview/AzureTrafficCollector.json

```diff
@@ -0,0 +1,989 @@
+      },
+      "description": "Emission policy properties."
+    },
+    "IngestionSourcesPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "sourceType": {
+          "type": "string",
+          "description": "Ingestion source type.",
+          "enum": [
+            "Resource"
+          ],
+          "x-ms-enum": {
+            "name": "sourceType",
+            "modelAsString": true
+          }
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Resource ID."
+        }
+      },
+      "description": "Ingestion policy properties."
+    },
+    "IngestionPolicyPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "ingestionType": {
+          "type": "string",
+          "description": "The ingestion type.",
+          "enum": [
+            "IPFIX"
+          ],
+          "x-ms-enum": {
+            "name": "ingestionType",
+            "modelAsString": true
+          }
+        },
+        "ingestionSources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IngestionSourcesPropertiesFormat"
+          },
+          "description": "Ingestion Sources."
+        }
+      },
+      "description": "Ingestion Policy properties."
+    },
+    "CollectorPolicyPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "ingestionPolicy": {
+          "$ref": "#/definitions/IngestionPolicyPropertiesFormat",
+          "description": "Ingestion policies."
+        },
+        "emissionPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EmissionPoliciesPropertiesFormat"
+          },
+          "description": "Emission policies."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state."
+        }
+      },
+      "description": "Collection policy properties."
+    },
+    "CollectorPolicy": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CollectorPolicyPropertiesFormat",
+          "description": "Properties of the Collection Policy."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of Collection Policy in a Traffic Collector."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceReference"
+        }
+      ],
+      "description": "Collection policy resource."
+    },
+    "Resource": {
```

Comment:
>"Resource" [](http://example.com/codeflow?start=4&length=10)

why not use the common type for TrackedResource instead?

Reply:
Used TrackedResource